test/integration/main_test.go

```diff
@@ -79,6 +84,16 @@ func clusterOperatorInsights() *configv1.ClusterOperator {
 	return operator
 }
 
+
+func clusterOperatorMonitoring() *configv1.ClusterOperator {
```

Comment:
I'd suggest to merge it with `clusterOperatorInsights` and create one function that can check any operator

Reply:
I have extracted function that can check any operator, it looks cleaner, but kept the functions, so the strings of operator names don't have to be used in code, but keeping the functions might be useless (for now I at least don't touch other tests)